sklearn/linear_model/ridge.py

```diff
@@ -1249,6 +1258,15 @@ class RidgeCV(_BaseRidgeCV, RegressorMixin):
     alpha_ : float
         Estimated regularization parameter.
 
+    Examples
+    --------
+    >>> from sklearn.datasets import load_diabetes
+    >>> from sklearn.linear_model import RidgeCV
+    >>> X, y = load_diabetes(return_X_y=True)
+    >>> clf = RidgeCV(alphas=[1e-3, 1e-2, 1e-1, 1]).fit(X, y)
```

Comment:
It might be better to keep consistent within your PR. (either use this one or things like ``alphas=[1, .01, .001, .0001]``)

Reply:
Done!